io.github.fheroes2.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: io.github.fheroes2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --filesystem=home
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+command: fheroes2
+modules:
+  - name: fheroes2
+    buildsystem: cmake
+    config-opts:
+        - "-DSDL2MAIN_LIBRARY="
+    sources:
+      - type: git
+        url: https://github.com/ihhub/fheroes2
+        branch: master
+        commit: 4ce7b1148b0c2ec4e5da67aeb79522942c7d6a95
+      - type: file
+        path: io.github.fheroes2.metainfo.xml
+      - type: file
+        path: AppIcon128.png
+      - type: file
+        path: io.github.fheroes2.desktop
+    post-install:
+       - install -Dm644 AppIcon128.png /app/share/icons/hicolor/128x128/apps/fheroes2.png
+       - install -Dm644 io.github.fheroes2.metainfo.xml /app/share/metainfo/io.github.fheroes2.metainfo.xml
```

Comment:
```suggestion
       - install -Dm644 io.github.fheroes2.metainfo.xm -t /app/share/metainfo/
```

Reply:
I've removed this lines. Okay?

Reply:
not sure you understood what I was suggesting here.